content/blog/2018-10-doc-blitz.md

```diff
@@ -0,0 +1,113 @@
++++
+date = "2018-10-04"
+title = "Announcing the Tokio Doc Blitz effort (we need you!)"
+description = "October 04, 2018"
+menu = "blog"
+weight = 990
++++
+
+In the past, there has been reoccurring feedback that Tokio is hard to
+understand. I believe a lack of good documentation plays a significant part.
+It's time to fix this problem.
+
+And because Tokio is open source, it is on you (the community) to make this
```

Comment:
maybe changing "you (the community)" to "us (the community)" might be better. Don't want anyone to get the idea you're not going to participate yourself.

Reply:
Good call 👍